doc/lsst.ap.pipe/apdb.rst

```diff
@@ -23,13 +25,13 @@ This page provides an overview of how to use |make_apdb|.
 Configuring the database
 ========================
 
-|ap_pipe| includes information about the database location and schema in its `ApPipeConfig`, and most users pass this information to the script through the :option:`--config <ap_pipe.py --config>` and :option:`--configfile <ap_pipe.py --configfile>` command-line options.
+The database is configured using an instance of `~lsst.dax.apdb.ApdbConfig`, which is made accessible through `ApPipeConfig`'s ``diaPipe.apdb`` option.
+|ap_pipe| command line users can pass configuration information to the script through the :option:`--config <ap_pipe.py --config>` and :option:`--configfile <ap_pipe.py --configfile>` command-line options.
```

Comment:
I think it's important to make it clear from the first sentence that this paragraph is talking about how the database is configured in the context of `ap_pipe.py` (which the second paragraph then extends to `make_apdb.py`). The fact that these two scripts share a config is very useful but potentially very confusing.

How about simply removing the passive voice? "|ap_Pipe| configures the database using an instance..."

Reply:
I'm confused about the point here. Surely `ApPipeTask` is the fundamental construct? Under what circumstances would I be using `ApPipeTask` but *not* configuring the database through `ApPipeConfig.diaPipe.apdb`? I'm worried that I'm being slow to catch on! :-)

Reply:
`ApPipeTask` _is_ the fundamental construct, but this page is specifically about how to use `make_apdb.py`, not `ApPipeTask`. Mentioning the latter at all, while necessary to explain the shared UI, introduces some mental friction, or at least it has in the past for other AP developers.